sdk/src/signer/presigner.rs

```diff
@@ -0,0 +1,80 @@
+use {
+    crate::{
+        pubkey::Pubkey,
+        signature::Signature,
+        signer::{Signer, SignerError},
+    },
+    thiserror::Error,
+};
+#[derive(Clone, Debug, Default)]
+pub struct Presigner {
```

Comment:
Do you mind adding a doc comment explaining this struct? (similar to NullSigner)